packages/mobile/src/home/WalletHome.tsx

```diff
@@ -159,8 +149,9 @@ export class WalletHome extends React.Component<Props> {
 
     return (
       <SafeAreaView style={styles.container}>
-        <DrawerTopBar />
+        <DrawerTopBar middleElement={<Logo />} scrollPosition={this.scrollPosition} />
         <AnimatedSectionList
+          scrollEventThrottle={50}
```

Comment:
Reading up on this with `react-native-reanimated`, it looks like it's required to declare a value on iOS, otherwise the animation can lag behind, but I think the actual value is ignored.
But what do you think of changing it to `scrollEventThrottle={1}` to clearly indicate we want the animation to be truly in sync with the scroll movement?

Reply:
AFAIK value is in ms, most examples use 16 or 1, so I'm a bit confused here. I'm proposing to use here 16 (corresponds to 60+ fps). WDYT?